### ISSUE TITLE
How to stop button text from being truncated

### ISSUE THREAD

**Issue by Jake-Mulhern** [AdelRedaa97/react-native-select-dropdown#105]
I have set up my drop down component, however the text within the dropdown is truncated.  I would like it to display the entire text regardless of the length of the string.

This is my current code:
```
<SelectDropdown
                defaultValue={Constants['walletList'][defaultDropdownValue().nickName] ? Constants['walletList'][defaultDropdownValue()].nickName : Constants['walletList'][defaultDropdownValue()]}
                data={Constants['walletList']}
                onSelect={(selectedWallet) => {
                  setGlobalVariableValue({
                    key: 'currentWallet',
                    value: selectedWallet
                  });
                }}
                buttonTextAfterSelection={(selectedWallet) => {
                  return selectedWallet.walletInstance.address
                }}
                rowTextForSelection={(item, index) => {
                  return item.walletInstance.address
                }}
                buttonStyle={{ backgroundColor: theme.colors['Background'], padding: 0, margin: 0 }}
                buttonTextStyle={{ fontSize: 14, padding: 0, margin: 0 }}
                renderDropdownIcon={() => {
                  return (
                    <Icon
                      size={18}
                      name={'AntDesign/down'}
                      color={'#3D4C63'}
                    />
                  )
                }}
              />
```

**Comment by Jake-Mulhern** [comment]
There also appears to be some padding or margin on the button.  I tried setting the buttonStyle to {{ margin: 0, padding: 0 }} but I still can't line up another element directly next to the text.  Is there any additional documentation on styling the button so that I can customize it?

**Comment by Juanfe98** [comment]
I'm not sure is this works for you but if it's the content inside the options... You can use renderCustomizedRowChild to render your own component and avoid the value to be truncated.

**Comment by AdelRedaa97** [comment]
Fixed in v4.0 take a look at the changes made in v4.0
https://github.com/AdelRedaa97/react-native-select-dropdown?tab=readme-ov-file#-major-changes

Feel free to open the issue again if it still exists
